baybe/constraints/discrete.py

```diff
@@ -42,6 +44,24 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:  # noqa: D102
         res = reduce(_valid_logic_combiners[self.combiner], satisfied)
         return data.index[res]
 
+    def to_polars(self) -> pl.Expr:
+        """Translate the constraint to Polars expression for filtering.
+
+        Returns:
+            The Polars Expr object to pass as an argument to filter().
```

Comment:
Would prefer ``pl.Expr`` in the docstring

Reply:
Same applies to some other docstring, won't comment it everywhere.

Reply:
changed